pt_miniscreen/app.py

```diff
@@ -94,25 +94,37 @@ def handler():
         return handler
 
     def handle_select_button_release(self):
+        if self.root.is_project_page:
+            return
+
         if self.root.can_enter_menu:
             return self.root.enter_menu()
 
         if self.root.can_perform_action:
             return self.root.perform_action()
 
         if self.root.can_select_row:
-            return self.root.enter_selected_row()
+            self.root.enter_selected_row()
+            self.root.start_project()
+            self.root.wait_project()
+            return self.root.exit_menu()
```

Comment:
i feel like too much of the project page's business has leaked into app & root here.

i think we should leave this button handler as just `enter_selected_row()`, have root push the project page, and then have the ProjectPage do the start & wait (can it do this on init? does it need a thread?). then it could ask root to exit_menu.

also, maybe root should have a method `is_navigation_disabled` instead of `is_project_page`. but I think that part we can leave for now and maybe revisit once we do the project stopping functionality